### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by jrhoun**
Hey Mike - 

Pull has my suggestions for changes. Main highlights:

1. Condensing the 2 intros to segmentation and personalization into one intro to help the section feel more cohesive.
2. Clarification requests for the Analytics Cloud related article
3. Wordsmithing

See comments as well in some other articles. Thank you!

### CODE REVIEW

File: docs/dxp/7.x/en/site-building/05-personalizing-site-experience/01-segmentation/assigning-roles-to-user-segments.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Assigning Roles to User Segments
 
-Since Liferay Portal CE 7.3 SP2, you can assign User Segments to Roles. Follow these steps:
+Since Liferay Portal CE 7.3 GA1, you can assign User Segments to Roles <!-- Should this be: "you can assign roles to user segments"? This task enables users to make it so anyone who has role [x] is part of segment [y], is that right? -->. Follow these steps:
```
Comment (mwilliams2014) [inline docs/dxp/7.x/en/site-building/05-personalizing-site-experience/01-segmentation/assigning-roles-to-user-segments.md:4]: Yes, this is assigning Roles to a User Segment, but it doesn't automatically assign someone who has that role to the User Segment.

File: docs/dxp/7.x/en/site-building/05-personalizing-site-experience/introduction-to-personalizing-site-experience.md [hunk 2]
```diff
@@ -17,11 +17,11 @@ See [Creating and Managing a Segment] to get started.
 
 ### Advanced and Segmentation with Liferay Analytics Cloud
 
-Supercharge your Segments by integrating your Liferay DXP instance with Anaytics Cloud. Analytics Cloud enables you to integrate DXP with Salesforce(tm) and other data sources to provide even more ways to understand and segment your user base.
+Supercharge your Segments by integrating your Liferay DXP instance with Liferay Analytics Cloud. Analytics Cloud enables you to integrate DXP with Salesforce(tm) and other data sources to provide even more ways to understand and segment your user base.
 
 <!-- screenshot of AC view -->
 
-Integration with AC also provides in-depth (ML powered?) data analytics capabilities to help you understand user behavior and session data for every page of your DXP site.
+Integration with Analytics Cloud also provides in-depth (ML powered?) data analytics capabilities to help you understand user behavior and session data for every page of your DXP site.
```
Comment (mwilliams2014) [inline docs/dxp/7.x/en/site-building/05-personalizing-site-experience/introduction-to-personalizing-site-experience.md:24]: @jrhoun what is ML powered?

Comment (jrhoun) [reply docs/dxp/7.x/en/site-building/05-personalizing-site-experience/introduction-to-personalizing-site-experience.md:24]: Me wondering if there's any machine learning elements on the Analytics Cloud side that are in use to help folks analyze / understand the data they get from integration. I don't know if that's the case, but thought it'd be a relevant place to mention that IF there is.

Comment (mwilliams2014) [reply docs/dxp/7.x/en/site-building/05-personalizing-site-experience/introduction-to-personalizing-site-experience.md:24]: Ah, ok. I'll try to get some more information on that.

Comment (mwilliams2014) [reply docs/dxp/7.x/en/site-building/05-personalizing-site-experience/introduction-to-personalizing-site-experience.md:24]: I found some webinars on the subject, so it looks like you can use Machine Learning with analytics and personalization. I'm not sure how fully integrated that is though.

**Comment by mwilliams2014** [comment]
Merged locally. Closing this and resending in a new PR since there are merge conflicts in your branch.

**Comment by mwilliams2014** [comment]
Sent updates in https://github.com/jrhoun/liferay-learn/pull/223

**Pull request closed without merging**